### PR TITLE
Ardupilot ap hal chibi os podman

### PR DESCRIPTION
**Pull request by burgeruser**
AP_HAL_ChibiOS: add podmanF405&F407&H743
This is the new ChibiOS Board

### CODE REVIEW

**Comment by khancyr** [comment]
Hello, 

Thanks for this. Same as the other boards, we require small readme with the  board informations and where to get it. Thanks

File: libraries/AP_HAL_ChibiOS/hwdef/podmanF405/hwdef-bl.dat [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# hw definition file for processing by chibios_pins.py
+# for minimal F405 bootloader
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# board ID for firmware load
+APJ_BOARD_ID 876
```
Comment (peterbarker) [inline libraries/AP_HAL_ChibiOS/hwdef/podmanF405/hwdef-bl.dat:8]: These boards have to be reserved with an entry in `Tools/AP_Bootloader/board_types.txt`.

Please make a separatre PR to reserve those IDs, then use those IDs in these files.  Note the ID should be the net available from the ArduPilot rnage, which starts at 1000 (so roughly 1115)

**Review by peterbarker — CHANGES_REQUESTED** [review]
There seems to be a lot of "todo" stuff in your H743 hwdef - were you planning on coming back to that?

**Comment by burgeruser** [comment]
I closed this Pull request.
changed into [#24161](https://github.com/ArduPilot/ardupilot/pull/24161)

File: libraries/AP_HAL_ChibiOS/hwdef/podmanF405/hwdef-bl.dat (hunk 1 above)
Comment (burgeruser) [reply libraries/AP_HAL_ChibiOS/hwdef/podmanF405/hwdef-bl.dat:8]: > These boards have to be reserved with an entry in `Tools/AP_Bootloader/board_types.txt`.
> 
> Please make a separatre PR to reserve those IDs, then use those IDs in these files. Note the ID should be the net available from the ArduPilot rnage, which starts at 1000 (so roughly 1115)

I closed this Pull request.
changed into [#24161](https://github.com/ArduPilot/ardupilot/pull/24161)

**Pull request closed without merging**